cmd/bluefin/main.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/blinklabs-io/bluefin/internal/config"
 	"github.com/blinklabs-io/bluefin/internal/indexer"
 	"github.com/blinklabs-io/bluefin/internal/logging"
+	"github.com/blinklabs-io/bluefin/internal/state"
```

Comment:
Can we call this persistence or storage? The "State" name is used within the reference node and I'd prefer avoid confusion.